app/presenters/formats/place_presenter.rb

```diff
@@ -0,0 +1,46 @@
+module Formats
+  class PlacePresenter < EditionFormatPresenter
+  private
+
+    GOVSPEAK_FIELDS = %i(
+      introduction
+      more_information
+      need_to_know
+    ).freeze
+
+    def schema_name
+      'place'
+    end
+
+    def document_type
+      'place'
+    end
+
+    def details
+      details = { external_related_links: external_related_links }
+
+      details = details.merge(govspeak_fields)
```

Comment:
It's a little hard to follow what keys this adds to the details hash, since the implementation is scattered through the whole class.  Could it be structured so that the keys are inline in this method to make it more obvious what it returns?